images-list

```diff
@@ -77,14 +82,18 @@ istio/proxyv2 rancher/istio-proxyv2 1.8.3
 istio/proxyv2 rancher/mirrored-istio-proxyv2 1.8.3
 jaegertracing/all-in-one rancher/jaegertracing-all-in-one 1.20.0
 jaegertracing/all-in-one rancher/mirrored-jaegertracing-all-in-one 1.20.0
+jenkins/jnlp-slave rancher/mirrored-jenkins-jnlp-slave 3.35-4
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.2.1
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.5.0
 jettech/kube-webhook-certgen rancher/mirrored-jettech-kube-webhook-certgen v1.2.1
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.3.0
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.4.0
 jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.3.0
 jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.4.0
+k8s.gcr.io/dns/k8s-dns-node-cache rancher/mirrored-k8s-dns-node-cache 1.15.13
+k8s.gcr.io/dns/k8s-dns-node-cache rancher/mirrored-k8s-dns-node-cache 1.15.7
```

Comment:
Not sure what to do with org for these kube ones.

Reply:
Same as above, I think the gcr ones tend to be this way because it's not a traditional registry like dockerhub, the org is kind of implied.